### ISSUE TITLE
The elected bodies should use the same mechanism for chair selection

### ISSUE THREAD

**Issue by hober** [w3c/process#891]
There are many, seemingly arbitrary inconsistencies between the methods used to [select the AB Chair(s)](https://w3c.github.io/process/#ABParticipation)
and to [select the TAG Chair(s)](https://w3c.github.io/process/#tag-participation). Absent compelling rationale, these should be made consistent.

**Comment by fantasai** [comment]
They're different because the bodies wanted different things from the chair selection mechanisms. If the TAG wants to change theirs, we can take a change it?

**Comment by martinthomson** [comment]
I see fixes from multiple directions:

1. The AB chair might be selected from outside the AB.  The Team chooses the chair with a 2/3 ratification vote from the AB.
2. The TAG chair has to be on the TAG.  The TAG chooses a chair by consensus.  Failure to select a chair leads to a Team choice.

The other aspects are all fine.

My preference is probably, in order:

1. A simple ballot.
2. Consensus, followed by a simple ballot.
4. Anything else.

I can see why the process has these differences.  The AB doesn't have a Team appointment or similar mechanism that can be used to compensate for the elected members being all unwilling to serve as chair.  And, in general, having the Team appoint a chair is potentially a way to avoid dysfunction of that sort.  However, I would say that groups operating at this level should be able to self-organize.

I list a simple ballot as preferable, because consensus in these bodies tends to lead to inertia.  People are unwilling to be seen to challenge established (and perhaps amply effective) leadership.  Requiring a ballot at the start of a new term creates a way to reaffirm a choice without prejudice, or -- in the finest of democratic tradition -- overthrow established leadership in a very civil and respectful manner.

My experience with a similar process (the IAB) showed that this is often largely an affirmation of what consensus had already established informally, but I have observed a few "challenges" to that consensus.  Challenging was generally encouraged. Challengers were regarded as a healthy sign that the body had multiple people willing to serve; it also created a safe space to discuss what the body expected from its leaders (I don't recall the TAG having that discussion, though I missed the first TAG offsite).

**Comment by cwilso** [comment]
With my individual-participant-of-the-AB hat on:

- The AB chair's selection by the Team is probably an artifact of history that isn't necessary to maintain.  The AB chair should be someone who can work with the Team effectively, but I don't think that's as necessary as "the AB is happy with their chairing."  Simple balloting by the AB itself would be fine.  Having been through this several times, I would actually prefer 2 - consensus, followed by simple ballot.  I was disappointed this was not simply a conversation in the AB followed by confirmation, tbh.  I do agree with @martinthomson that groups at this level should be able to self-organize.
- the AB could, however, end up with no one willing to or proficient at chairing.  The TAG could solve this problem by appointment; the AB chose to solve it by enabling the possibility of external chairing.  In practice, this has not been used, but I think it's a good idea to do so (the TAG did use its appointment seats to ensure good chairs for years).  There are two ways to solve this: have a shared process for selecting chairs that allows the groups to appoint outside Chairs, or enable appointments for the AB.  I think the former would be easier to add, and infrequently used, so I'd probably suggest that.
- There is a small subtlety to the AB process that bears highlighting: the chair gets to select their co-chair.  In practice, it's important to recognize that not every good chair would work well with every other chair.  That could just fall into the category of "self-organize".

**Comment by fantasai** [comment]
Chairing discussions can feel confrontational, particularly in the case where some members might feel that a change in chairs is warranted; or if they have reservations about proposed chairs. For example, opening a discussion with something like "I propose this other person, does anyone object?", especially if either the proposer or the proposed candidate is the current chair, doesn't leave a lot of room for people uncomfortable with the proposal to say anything about it. Less confident or more reticent members definitely will not say anything, and it also doesn't exactly entice other people to volunteer.

So I think there is something useful to the Team's role as a buffer here in the AB procedures that we might want to preserve, though I agree that if there is consensus in the body the Team shouldn't have the discretion to appoint someone else.

Bottom line, I think we can improve the Process here, but we need to be mindful of these things as well.

**Comment by cwilso** [comment]
I believe the Membership would be best served by using the same chair process across the elected bodies - or at least across the AB and the TAG.  (The Board of Directors, of course, has this process defined in the bylaws.  It's a consensus election - section IV.18.)

This would leave two options, with an optional addition:

1. A la the TAG's current situation, "Participants in the [TAG](https://www.w3.org/policies/process/#technical-architecture-group) choose by consensus their Chair or co-Chairs; in the absence of consensus, the [Team](https://www.w3.org/policies/process/#team) appoints the Chair or co-Chairs of the TAG. The Chair or co-Chairs must be selected from the participants of the TAG."  In short: elected body members are expected to figure out how to do this.  If this is done consistently, yearly, it should be a reproducible routine that doesn't cause challenges.
2. A la the AB's current situation, "With the input of the [AB](https://www.w3.org/policies/process/#advisory-board), the [Team](https://www.w3.org/policies/process/#team) appoints the Chair, who should choose a co-chair among the elected participants. The Chair(s) are subject to ratification by secret ballot by two thirds of the AB upon appointment."  In practice, this has required a two-step procedure: essentially an election of the chair, followed by ratification of the chair and their chosen co-chair.

What has landed us in a less comfortable situation in the AB right now is that if the chosen co-chair - i.e., not the person originally chosen by the Team with input from the AB - steps down, the remaining (appointed) chair of the AB has to either solo-chair, or they have to start from scratch with the two-step routine from above.  As the Team *OR* a majority of AB members could trigger a full selection at any time, this seems like overkill, when the original chair was chosen by that very body, less than a year before; they're only asking for re-ratification with a new co-chair.

My personal belief is that the "with input from" and "secret ballot" portions of this process have actually led to more discontent and damaged credibility than would have occurred if members had simply been transparent about their desires for chairing to begin with.  Chair ratifications in the past have been followed by negative statements by members about the chairs chosen; those statements should have fed into EVERY member's ratification decision, not been an after-the-fact credibility question.

In the case of the AB and the current situation, I disagree with @fantasai's statement because I was _**always**_ the "co-chair" - the one chosen by the Chair who was appointed, and tolerated in the ratification, not selected by my colleagues. I'm okay with that - but then we should have given the appointed chair the ability to replace me without starting over, as long as they were approving this.  With that context, the Chair proposing someone else they would be comfortable co-chairing with for ratification by the rest of the group should not have been uncomfortable - it's literally the way it's done in the middle of the routine every year.  (After the Team chooses their appointment, the Chair-appointee says "what do you all think about so-and-so co-chairing with me?"  The AB doesn't get to weigh in, just object.)

So in short: I would suggest either a) leave things the way they are on the AB (in which case, the TAG should change for symmetry and getting all the good effects that process claims to have), but adding a caveat that if the non-appointed co-chair steps down, the appointed chair can re-select a co-chair with ratification, or b) switch to the TAG's process, which basically says "everyone be adults, choose a chair by consensus" which ensures people share how they really feel.

**Comment by frivoal** [comment]
> the "co-chair" - the one chosen by the Chair who was appointed, and tolerated in the ratification, not selected by my colleagues

That's an unfortunate dynamic, and I'm sorry I hadn't realized earlier it could be perceived this way. I think the main goal of this pattern was to enable yet constrain the ability to have one of the co-chairs be a non-member of the AB. But introducing a difference in how real of a chair you are seems unintended and undesirable. If we keep the team intermediation, I'd be tempted to replace what we have by a proposal of one or two co-chairs as a single ticket, meant to complement or reinforce each other, rather than a "real" chosen chair and a tolerated "sub"-chair.

As for whether we need the intermediation of collect feedback / propose / secret ballot provided by the Team or not, I think that optimizes for different things.
* intermediation should enable picking a preferred set of chairs over an acceptable-yet-not-preferred one who would be the default choice, because it enables expressing that preference without having to be openly against anyone.
* a direct approach of talking through it without intermediation favors continuity: as long as the default choice of chairs remains at least somewhat acceptable, it is hard to push back an propose anything else, even if the alternative would actually have more support. But we do gain stability.

**Comment by fantasai** [comment]
This is my proposed replacement for the current Process, for both groups:

> With the advice of the participants and the consent of the candidates, and abiding by the expressed consensus of the group if any, the Team appoints one or two Chair(s), at least one of whom must be a participant of the body. The Chair(s) are subject to ratification by secret ballot, requiring approval by two thirds of the elected participants. 

This prevents the Team appointing contrary to the consensus of the group; but enables them to intermediate (subject to ratification) when there isn't one. It keeps the two co-chairs on an even level (which I strongly prefer also), while also requiring them to agree to work together. And it enables the use of a balloting process, if the group decides to set one up. (I don't want to require it in the Process at this point because I think we need some experimentation.)

Possible ways this could run:
- The chairs confront the group and propose their preferred set of chairs and ask for objections. The Team appoints accordingly. (This is not my preferred procedure, but it's possible. I would suggest discouraging this kind of thing in /Guide.)
- The group decides on a balloting process, and runs it, and then confirms the result as their consensus position. The Team appoints accordingly.
- The Team asks everyone to note if they would be willing to serve if the rest of the group elected them, participants forward their nuanced thoughts about the candidates to the Team, and the Team considers their input and appoints accordingly.
- The Team notes that none of the participants are interested in chairing and suggests an external chair. The participants discuss this among themselves and resolve to accept the suggestion (or a different suggestion). The Team appoints accordingly.

**Comment by frivoal** [comment]
The AB has had a discussion about this, which did not conclude yet. It seems to me that the AB was generally in favor of the direction of this proposal, but that there were arguments about some of the details which did not conclude yet.

(my understanding of) a few points which were raised in AB discussion about the proposal in the above comment:
* secret ballot has the downside that some information (both the existence of a negative vote, and the motivation for it) which would be useful to the selection process can remain hidden
* secret ballot in such small bodies can be quite hurtful for the candidates if they do not get approved and have no idea why
* on the flip side, expressing criticism in the open can be hard, and more so in certain cultures, so requiring them to be non secret may (differently) suppress information
* Ideally, the information, possibly with mediation by the Team, should come out earlier. But what if it doesn't?
* 2/3 of the participants (rather than voters) gets in the way of people deliberately abstaining, which is something to think about, and may have different implications in a secret vs open ballot.

**Comment by fantasai** [comment]
The secret ballot ratification step is meant to be a backstop: ideally the selection process is organized in a way that, by the time it's run, there's no opposition. But if the process was run poorly, or there's some other problem, it ensures that the chair selection actually has the acceptance of at least 2/3 of the body. I would be opposed to removing this step from the Process; instead the bodies should focus on improving their selection process (lowercase "p" ;) until the ratification step feels routinely redundant.

**Comment by plehegar** [comment]
my two cents: unless we identify a problem with the way the TAG has been choosing its Chairs, we should not change the Process on how TAG Chairs are picked. I like consistency but I also like fixing actual existing issues. The length of this issue just demonstrates how dangerous, and potentially damaging, this could become for the TAG.

**Comment by jyasskin** [comment]
IIUC, we haven't been through a TAG chair selection in most of a decade, so let's wait until the next TAG picks its chairs before deciding on any changes to our process. [Fantasai's proposed process](https://github.com/w3c/process/issues/891#issuecomment-2450848305) seems reasonable to me, and we can keep it in mind while picking chairs to imagine how it could change things.

**Comment by fantasai** [comment]
> Then let's change the AB's method to match the TAG's! :)

The AB, due to its role, has reasons to integrate with the Team that don't apply to the TAG, so I don't think simply copying over what we've codified for a functionally independent TAG makes sense.

**Comment by wareid** [comment]
Based on both personal experience as a chair and going through the motions now several times, I agree with much of what is being said here, and really strongly support anything we can do to simplify this process, and make it more consistent between the two bodies. 

We should be prioritizing a process where the body (TAG or AB) is encouraged first to find chairs from its membership, and should only require outside intervention in the case where either consensus cannot be reached or the body requires additional support to meet their needs. We should also be encouraging these discussions to be open and within the body, I think it is really important to remember that the groups are small, and if we cannot have frank discussions about something as important as chairing, what hope do we have when it comes to more serious matters? 

I wonder if a hybrid of the AB/TAG chairing policies can be reached, something like:
1. Participants in the [elected body] should attempt to find chairs first from within the elected body through discussion and consensus. If there is aren't enough willing chair candidates, or a gap in skills/representation, the elected body can seek a chair from outside the body.
2. If a consensus cannot be reached within the elected body, the Team will work with the body to appoint chairs. 

The most radical part of this is removing the ratification step, which is only on the AB and I think is what has caused us the most grief. I think both simplifying the process to prioritize the group choosing its own chairs and having the Team intervene as a last resort (with their decision being final) will remove some of the grief, and encourage the elected bodies to take the initial step much more seriously. I also think giving the TAG the AB's option to bring in an external chair is helpful too, while it should not be the first option, it gives both bodies the freedom to seek out chairs in cases where there is a lack of representation or willing candidates.

**Comment by cwilso** [comment]
I would go a bit further and say the thing that has the most grief in the AB isn't specifically the ratification step, but the use of a secret ballot.  This has repeatedly caused the AB to _**not**_ have an open and honest discussion and consensus-building on chairing.

(The TAG has the AB's ability to bring in an external chair, they would just have to use an appointment slot to do it.)

**Comment by frivoal** [comment]
Based on the latest discussions in the AB, I would say that there is agreement that the current AB chair selection mechanism isn't right, but there is not yet agreement about what to replace it with.

**Comment by jyasskin** [comment]
We're not quite finished with this year's TAG chair selection, but some thoughts so far:

* We made some mistakes in how we ran the selection this year, which I think we can just fix next year. It would be good to document best practices in a less-formal and more-evolvable place than the Process. I've started that for the TAG in https://github.com/w3ctag/process/pull/39, and we should share notes with at least the AB and possibly the Board.
  * I'm hopeful this can resolve the [concerns](https://github.com/w3c/process/issues/891#issuecomment-2677280300) about having to raise negative feedback about chair candidates while looking for consensus: if there's a non-candidate coordinator, they can receive the negative feedback and help express it in a constructive way, without "exit" being the only option, in the terminology of [Exit, Voice, and Loyalty](https://en.wikipedia.org/wiki/Exit,_Voice,_and_Loyalty).
* I think a voting-based backstop is a good idea between the attempt at consensus and having the Team pick, but I'd like the Process to keep encouraging us to start with an attempt at consensus. (That is, [Martin's choice 2](https://github.com/w3c/process/issues/891#issuecomment-2398865855).) If we'd started with voting this year, I think that we would have failed to progress the issues that we're hoping to fix next year.
  * It's true that a body can agree that they'll vote and then treat the outcome as consensus. (Which reminds me of a bit in Vernor Vinge's _Deepness in the Sky_ about democracy being a "pre-tech" consensus system.) But it'll be easier to do this if it's endorsed by the Process.
  * Votes should be among slates of chairs, rather than individual chairs, where candidates would have to agree to be on any slates they're listed on and could appear on multiple slates. Slates could even be more complicated than simple co-chairing situations, for example to allocate particular parts of the chairing role to particular people, but I wouldn't want the Process to guess too much at the possible details there.

**Comment by frivoal** [comment]
@cwilso the first paragraph in your https://github.com/w3c/process/issues/891#issuecomment-2679340949 comments seems to be an answer to #921 rather than to this issue.

**Comment by cwilso** [comment]
@frivoal My first paragraph was in response to @chaals' third paragraph in https://github.com/w3c/process/issues/891#issuecomment-2678607478.
